src/core/Workspace.ts

```diff
@@ -44,11 +45,19 @@ export default class Workspace {
       workspace.config.data.kong_admin_url = process.env.KONG_ADMIN_URL
     }
 
-    if (process.env.KONG_ADMIN_TOKEN) {
+    if (process.env.KONG_ADMIN_TOKEN_FILE) {
+      if ((await this.exists(process.env.KONG_ADMIN_TOKEN_FILE)) === false) {
+        throw new Error()
```

Comment:
Can we make the error if there is a KONG_ADMIN_TOKEN_FILE, but the file does not exist more verbose?

Reply:
@HenriPro see latest push.